[DOCS] Upsert documentation clarification (#120684)
elastic/elasticsearch: docs/reference/docs/update.asciidoc

[discrete]
===== Upsert

> Upserts are a powerful operation that allows you to perform both update and insert actions in a single request. This is particularly useful when you want to update a document if it exists, or insert a new document if it does not.

Reviewer: ```suggestion
An upsert operation lets you update an existing document or insert a new one if it doesn't exist, in a single request.
```

 the second sentence just restates the first in different words.
